ghost/collections/src/CollectionsService.ts

```diff
@@ -183,6 +184,10 @@ export class CollectionsService {
         });
 
         this.DomainEvents.subscribe(PostEditedEvent, async (event: PostEditedEvent) => {
+            if(isEqual(event.data.current, event.data.previous)) {
```

Comment:
Can we make this more specific? I think we just need to check `published_at`, `featured` and `tags`!

Reply:
The fields on the current/previous are pretty [limited already](https://github.com/TryGhost/Ghost/blob/78e2cb0c284dceca3803700fa31bff559dc41213/ghost/collections/src/events/PostEditedEvent.ts#L1-L19). Or do you mean to future proof once we add more fields to the event itself?

Reply:
Yeah I think just to be future proof/more specific 🤷  NABD but will save unnecessary updates